### ISSUE TITLE
" " or ' ' not working in exec key(s)

### ISSUE THREAD

**Issue by salkin-mada** [kyoheiu/felix#180]
Thanks for a fantastic TUI filemanager!! <3
I just started using it and found the following kinda puzzling.
```
exec:
  'kitty --detach --hold nvim':
    [scd, sc, cpp, hpp, lua, md, txt, scad]
```
or lets say I want to use `imv -f` / `feh -.` or any other command with some _options_.
I would expect this to work.

**Comment by kyoheiu** [comment]
Currently options are not supported. I'll check if I can handle it.

**Comment by kyoheiu** [comment]
Ok, implemented (very naively) handling options by c4e8e35c80b0fed2a90d349a6059c7b0cbe9d135 .
(Currently only `Enter | l | Right arrow` works, 'o' does not work yet)
It'd be great if you install it via develop branch and check if this works.

**Comment by salkin-mada** [comment]
Now using 'cmd --flag' as key does work (with `Enter | l | Right arrow`.) But it always opens the first image in a directory.

**Comment by salkin-mada** [comment]
Also there seems to be some restrictions in regard to what chars can be used?
`mpv --fullscreen=no` on **[mp4]** as an example, fails miserably.

**Comment by kyoheiu** [comment]
Haha, I missed the important line: Fixed with 9f4b3b3173bd2e9f2262c8640c83f8cf3bf8f496, should work now.
This is a really useful feature, thanks for the suggestion. I will fully implement this and include this in the next release.

**Comment by kyoheiu** [comment]
> Now using `cmd --flag` as key does work (with `Enter | l | Right arrow`.) But it always opens the first image in a directory.

Haha, I missed the important line: Fixed about with 9f4b3b3173bd2e9f2262c8640c83f8cf3bf8f496, should work now.
This is a really useful feature, thanks for the suggestion. I will fully implement this and include this in the next release.


> Also there seems to be some restrictions in regard to what chars can be used? `mpv --fullscreen=no` on **[mp4]** fails.

I'll check this.

**Comment by kyoheiu** [comment]
Hm, in my environment 'mpv --fullscreen=no' somewhat works, with a little bit buggy screen though.

![screenshot](https://user-images.githubusercontent.com/61998590/216790833-b4f632e6-6790-4029-a6b7-cde26e2de206.png)
![executes](https://user-images.githubusercontent.com/61998590/216790836-630b22a6-0d0d-4704-99a0-583adf0876bc.png)

The key is split by ascii-whitespace, so chars like `-` or `=` has no effect to the result.

**Comment by kyoheiu** [comment]
Created PR:  #182

**Comment by kyoheiu** [comment]
Released by v2.2.5.

**Comment by salkin-mada** [comment]
nice!
